pkg/config/config.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+
+	"gopkg.in/yaml.v2"
+)
+
+type Configurer interface {
+	GetSynonymCategory(category string) string
+}
+
+type Synonym struct {
+	Categories map[string]string
+}
+
+func New(fileName string) (*Synonym, error) {
+	file, err := os.Open(fileName)
+	if err != nil {
+		return nil, errors.Wrapf(err, "couldn't open file: %s", fileName)
+	}
+	defer file.Close()
+	var synonym Synonym
+	err = yaml.NewDecoder(file).Decode(&synonym)
```

Comment:
shouldn't you store all lowercase as well?

Reply:
words should be in lowercase already in file